code/handler.py

```diff
@@ -1,54 +1,50 @@
 import json
 import boto3
+import datetime
 from botocore.config import Config
 
 
 def lambda_handler(event, context):
     dry_run = get_dry_run_parameter(event)
     regions = get_regions_parameter(event)
+    delete_only_if_created_before = datetime.datetime.now(datetime.UTC) + datetime.timedelta(days=-3)
 
     for region in regions:
         print('REGION: {}'.format(region))
         config = Config(retries={'max_attempts': 50, 'mode': 'standard'})
         cfn_client = boto3.client('cloudformation', region_name=region, config=config)
 
-        failed_change_sets = find_failed_change_sets(cfn_client)
-        delete_change_sets(failed_change_sets, cfn_client, region, dry_run)
+        n_failed_change_sets = delete_failed_change_sets(cfn_client, delete_only_if_created_before, dry_run)
+        print('There were {} failed change sets in {}.'.format(n_failed_change_sets, region))
 
 
-def find_failed_change_sets(cfn_client):
-    failed_change_sets = []
+def delete_failed_change_sets(cfn_client, delete_only_if_created_before, dry_run):
+    n_failed_changesets = 0

```

Comment:
rename this variable to `deleted_failed_changesets_counter` or something similar